js/packages/openinference-instrumentation-langchain/tsconfig.json

```diff
@@ -2,9 +2,10 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "dist",
-    "rootDir": "."
+    "rootDir": ".",
+    "types": ["node", "jest"]
   },
   "files": [],
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "include": ["src/**/*.ts"],
```

Comment:
no type checking in the tests?

Reply:
i will add this back just removed cause tests were messy as i was working and it was annoying to fix types before i could run / debug